pkg/cmd/pipeline/start_test.go

```diff
@@ -1417,6 +1417,144 @@ func Test_start_pipeline_newGitRes_noExistingRes(t *testing.T) {
 	}
 }
 
+func Test_start_pipeline_newMultipleRes_noExistingRes(t *testing.T) {
```

Comment:
Thanks, @piyush-garg.
It would be really nice to have one more test with two different types of resource could be `git` and `image`.